drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/test/java/org/drools/workbench/screens/guided/dtable/client/widget/table/utilities/ColumnUtilitiesTest.java

```diff
@@ -125,6 +129,60 @@ public void testCanAcceptOtherwiseWrongOperator() throws Exception {
         assertFalse(ColumnUtilities.canAcceptOtherwiseValues(column));
     }
 
+    @Test
+    public void testGetValueConditionColumn() throws Exception {
+        column.setFactField(FIELD_NAME);
+        column.setFieldType("Integer");
+        column.setValueList("a,1,1.1, ,-1");
+        String[] valueList = utilities.getValueList(column);
+        assertEquals(2,
+                     valueList.length);
+        assertEquals("1",
+                     valueList[0]);
+        assertEquals("-1",
+                     valueList[1]);
+    }
+
+    @Test
+    public void testGetValueSetFieldColumn() throws Exception {
+        pattern.setFactType(FACT_TYPE);
+        pattern.setBoundName("$a");
+        when(model.getConditions()).thenReturn(Arrays.asList(pattern));
+
+        final ActionSetFieldCol52 column = new ActionSetFieldCol52();
+        column.setFactField(FIELD_NAME);
+        column.setBoundName("$a");
+        column.setValueList("a,1,1.1, ,-1");
+        when(oracle.getFieldType(FACT_TYPE,
+                                 FIELD_NAME)).thenReturn("Integer");
+
+        String[] valueList = utilities.getValueList(column);
+        assertEquals(2,
+                     valueList.length);
+        assertEquals("1",
+                     valueList[0]);
+        assertEquals("-1",
+                     valueList[1]);
+    }
+
+    @Test
+    public void testGetValueInsertFactColumn() throws Exception {
+        final ActionInsertFactCol52 column = new ActionInsertFactCol52();
+        column.setFactField(FIELD_NAME);
+        column.setFactType(FACT_TYPE);
+        column.setValueList("a,1,1.1, ,-1");
+        when(oracle.getFieldType(FACT_TYPE,
+                                 FIELD_NAME)).thenReturn("Integer");
```

Comment:
I try to use ```DataType.TYPE_NUMERIC_INTEGER``` which are the "internal" data-types used by the ```DataModelOracle``` rather then what could appear to be a "magic string".